src/main/java/fish/payara/extras/diagnostics/collection/collectors/HeapDumpCollector.java

```diff
@@ -132,7 +132,7 @@ public int collect() {
 
             RemoteCLICommand remoteCLICommand = new RemoteCLICommand("generate-heap-dump", programOptions, environment);
             String result = remoteCLICommand.executeAndReturnOutput();
-            LOGGER.info(result);
+            LOGGER.info("Heap Dump Generated");
```

Comment:
Surely this would remove any error messages, i.e. server is not running/has never been started and just output that the heap dump has been generated?